packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol

```diff
@@ -136,14 +130,15 @@ contract LiquidityProviderSampler is
         if (registryAddress == address(0)) {
             return address(0);
         }
+
         bytes memory callData = abi.encodeWithSelector(
-            ILiquidityProviderRegistry(0).getLiquidityProviderForMarket.selector,
+        ILiquidityProviderRegistry(0).getLiquidityProviderForMarket.selector,
             takerToken,
             makerToken
         );
         (bool didSucceed, bytes memory returnData) = registryAddress.staticcall(callData);
```

Comment:
Why no `try/catch` here?

Reply:
There's explicit tests for when the registry doesn't exist, where we want to fail silently. I believe we hit the case where the abi decode failed and reverted.